portfolio/src/main/webapp/style.css

```diff
@@ -0,0 +1,9 @@
+::root{
+    font-family:"Roboto";
+    font-style:normal;
+    font-size:5vw;
```

Comment:
keep the declarations in alphabetic order: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
Does the alphabetization apply to declarations starting with "--"?